static/js/modules/helpers.js

```diff
@@ -216,6 +215,41 @@ function zeroPad(container, item, appendee) {
   });
 }
 
+function amendmentVersion(data) {
+  if (data === true) {
+    return '<i class="icon-circle--check-outline--inline--left"></i>Current version';
+  }
+  else if (data === false) {
+    return '<i class="icon-circle--clock-reverse--inline--left"></i>Past version';
+  }
+  else {
+    return 'Version unknown';
+  }
+}
+
+function amendmentVersionDescription(row) {
+  var description = '';
+  var amendment_num = 1;
+
+  // because of messy data, do not show if not e-filing or null amendment indicator
```

Comment:
Thanks for the suggestion! I didn't want to nest much but didn't think of a cleaner way at the moment.